R/Solver.R

```diff
@@ -9,161 +9,197 @@
 ### or it can be a matrix of column vectors to solve for.
 ### If Rank_Analysis has been done, the rank_an output can be inputted to avoid repeating rank-finding.
 ### DFT_obj is an output from the gen_DFT() function
-solve_WA <- function(DFT_obj,A_mat,v,
-                     rank_obj=NULL,DID_full=FALSE) {
+solve_WA <- function(
+    DFT_obj,
+    A_mat,
+    v,
+    rank_obj = NULL,
+    DID_full = TRUE
+) {
```

Comment:
Single-indent function definition to clearly separate the function name from its arguments.